artifactory/commands/buildinfo/publish.go

```diff
@@ -91,6 +93,35 @@ func (bpc *BuildPublishCommand) Run() error {
 	if err != nil {
 		return err
 	}
+
+	buildTime, err := time.Parse(buildinfo.TimeFormat, buildInfo.Started)
+	if errorutils.CheckError(err) != nil {
+		return err
+	}
+
+	// Convert from nanoseconds to milliseconds
+	timestamp := buildTime.UnixNano() / 1000000
+	artVersioin, _ := servicesManager.GetVersion()
+
+	if artVersioin[0:1] == "6" {
```

Comment:
I suggest casting the value of ```artVersioin[0:1]``` into an int, so that we can check if the value is equals or smaller than 6. This way, this if statement. will also catch the case of Artifactory 5, 4 and 3.